daffodil-core/src/main/scala/org/apache/daffodil/core/runtime1/ModelGroupRuntime1Mixin.scala

```diff
@@ -41,7 +41,6 @@ trait ModelGroupRuntime1Mixin { self: ModelGroup =>
             case t: Term => t.termRuntimeData
           }
         }
-      case _ => Nil
```

Comment:
Should we remove the match and just do `val res = self.groupmembers.map { ... }`. The match is unnecessary since `this` is always a `ModelGroup`.